<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDirectionalLightWidget</class>
 <widget class="QWidget" name="QgsDirectionalLightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Azimuth</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsDoubleSpinBox" name="mAzimuthSpinBox">
     <property name="suffix">
      <string notr="true">°</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>359.899999999999977</double>
     </property>
     <property name="value">
      <double>315.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDial" name="mDialAzimuth">
     <property name="maximum">
      <number>3599</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchTarget">
      <double>60.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Altitude</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsDoubleSpinBox" name="mAltitudeSpinBox">
     <property name="suffix">
      <string notr="true">°</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="value">
      <double>45.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="mAltitudeSlider">
     <property name="maximum">
      <number>900</number>
     </property>
     <property name="value">
      <number>450</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
